src/elements/component-frame.js

```diff
@@ -9,14 +9,14 @@ export default class ComponentFrame extends HTMLElement {
 
     constructor() {
         super();
-
         this.iframe = document.createElement('iframe');
         this.iframe.setAttribute('frameborder', 0);
         this.iframe.setAttribute('style', IFRAME_STYLE);
-        this.iframe.setAttribute('sandbox', 'allow-scripts');
+        this.iframe.setAttribute('sandbox', 'allow-scripts allow-same-origin');
```

Comment:
this `allow-same-origin` was so I wouldn't have to jump through a bunch of CORS hoops right now. I'm not sure I want to leave it long term, as combined with `allow-scripts` it would allow components to break sandboxing.  But without out it, every network request is sent with a `null` origin, which makes every request cross-origin.